TRB_label_assist/performance_eval.py

```diff
@@ -274,6 +278,7 @@ def cv_for_all_algs(uuid_list,
                     min_samples=False,
                     raise_errors=False):
     cv_results = {}
+    pathlib.Path('first_trial_results').mkdir(parents=True,exist_ok=True) #needed first time
```

Comment:
would have been good to have included this change in the description of the related commit

https://github.com/humbleOldSage/e-mission-eval-private-data/commit/a34836faf364f9548d3ab5373c8efd0d1e729a53